src/Quests/CustomQuest.cs

```diff
@@ -72,6 +76,18 @@ internal void Complete(IQuestInfo questInfo)
             this.Completed?.Invoke(this, questInfo);
         }
 
+        internal void Accept(IQuestInfo questInfo)

```

Comment:
This method is not used. Use it in watchdog in the quest accept part.

Reply:
Does it mean this method should be moved to watchdog or just delete it?

Reply:
Just call it in watch dog.